cobbler/modules/managers/import_signatures.py

```diff
@@ -137,6 +145,30 @@ def get_file_lines(self, filename: str) -> Union[List[str], List[bytes]]:
             )
         return []
 
+    def get_file_version(self) -> List[int]:
```

Comment:
I would prefer to return a Tuple or NamedTuple as we can per certain that this method returns integers. Technically we don't need to use a Tuple but it would be faster afaik.